src/utils.ts

```diff
@@ -89,6 +89,15 @@ export function buildRequestApiOptions (sdk: GenesysCloudWebrtcSdk, opts: Partia
     opts.method = 'get';
   }
 
+  if (!opts.customHeaders && sdk._config.customHeaders) {
+    opts.customHeaders = sdk._config.customHeaders;
+  }
+
+  opts.customHeaders = {
```

Comment:
With this in place, I don't think we need the code above, but we can tweak that later.